scripts/write-changelog.ts

```diff
@@ -0,0 +1,26 @@
+import { readFile, writeFile } from 'node:fs/promises';
+import { exit } from 'node:process';
+
+async function writeChangelog(): Promise<void> {
+  if (!process.env.CHANGELOG) {
+    throw new Error('CHANGELOG environment variable not set.');
+  }
+  if (!process.env.VERSION) {
+    throw new Error('VERSION environment variable not set.');
+  }
+  const unifiedChangelog = await readFile('CHANGELOG.md', { encoding: 'utf8' });
+  await writeFile(
+    'CHANGELOG.md',
+    unifiedChangelog.split('\n').slice(0, 30).join('\n') +
```

Comment:
Can you explain this a bit?
This looks like the first (arbitrarily? 30 lines) are taken, put in front of the new changelog & version, and then the remaining lines are attached at the end??

Reply:
I can't really to be honest. I think this just takes out the comments that we have in the changelog file. Nobody is touching it manually anyways, so they are not helpful. I didn't touch this logic in this PR, only moved it to a different place.